test/powershell/Modules/Microsoft.PowerShell.Utility/ConvertTo-Html.Tests.ps1

```diff
@@ -136,8 +136,9 @@ After the object
         $returnString | Should Be $expectedValue
     }
 
-    It "Test ConvertTo-HTML meta with invalid properties should throw"{
-        { ($customObject | ConvertTo-HTML -Meta @{"authors"="John Doe";"keywords"="PowerShell,PSv6"}) -join $newLine } | Should Throw "authors is not a supported meta property. Accepted meta properties are content-type, default-style, application-name, author, description, generator, keywords, x-ua-compatible, and viewport."
+    It "Test ConvertTo-HTML meta with invalid properties should throw warning" {
+        ($customObject | ConvertTo-HTML -Meta @{"authors"="John Doe";"keywords"="PowerShell,PSv6"} 3>&1) -match "Accepted meta properties are" | Should Be $true
```

Comment:
Since the message could be localized, you probably just need to check that the warning contains the Meta tags you used in the test